packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -644,6 +645,15 @@ async function startCompassElectron(
     chromeArgs.push('--showed-network-opt-in=true');
   }
 
+  // Logging output from Electron, even before the app loads any JavaScript
+  const electronLogFile = path.join(LOG_PATH, `electron-${runCounter}.log`);
+  chromeArgs.push(
+    // See https://www.electronjs.org/docs/latest/api/command-line-switches#--enable-loggingfile
+    '--enable-logging=file',
+    // See https://www.electronjs.org/docs/latest/api/command-line-switches#--log-filepath
+    `--log-file=${electronLogFile}`
```

Comment:
Hmm. I wonder if we should add --log-level while at it.

Reply:
It's info by default and I've found that it's already pretty chatty.
Here's the list of available levels: https://source.chromium.org/chromium/chromium/src/+/main:base/logging.h;l=370-378?q=logging::LogSeverity&ss=chromium

Reading [the `--log-level` documentation](https://www.electronjs.org/docs/latest/api/command-line-switches#--log-leveln) it seems to me that one would need to play around with `--v` and `--vmodule` to get specific verbose logging and it's probably best to adjust on a case-by-case basis:
> Note that two complimentary logging mechanisms in Chromium -- LOG() and VLOG() -- are controlled by different switches. --log-level controls LOG() messages, while --v and --vmodule control VLOG() messages. So you may want to use a combination of these three switches depending on the granularity you want and what logging calls are made by the code you're trying to watch.
> 
> See [Chromium Logging source](https://source.chromium.org/chromium/chromium/src/+/main:base/logging.h) for more information on how LOG() and VLOG() interact. Loosely speaking, VLOG() can be thought of as sub-levels / per-module levels inside LOG(INFO) to control the firehose of LOG(INFO) data.
> 
> See also --enable-logging, --log-level, --v, and --vmodule.